Core/include/Acts/TrackFinder/CombinatorialKalmanFilter.hpp

```diff
@@ -1134,13 +1133,15 @@ class CombinatorialKalmanFilter {
   /// the track finding.
   ///
   /// @return the output as an output track
-  template <typename source_link_t, typename start_parameters_t,
+  template <typename source_link_container_t, typename start_parameters_t,
             typename comb_kalman_filter_options_t,
             typename parameters_t = BoundParameters>
-  Result<CombinatorialKalmanFilterResult<source_link_t>> findTracks(
-      const std::vector<source_link_t>& sourcelinks,
-      const start_parameters_t& sParameters,
-      const comb_kalman_filter_options_t& tfOptions) const {
+  Result<CombinatorialKalmanFilterResult<
+      typename source_link_container_t::value_type>>
+  findTracks(const source_link_container_t& sourcelinks,
+             const start_parameters_t& sParameters,
+             const comb_kalman_filter_options_t& tfOptions) const {
+    using source_link_t = typename source_link_container_t::value_type;
```

Comment:
Nitpicking: following the naming convention this should be `SourceLink`, I think.

Reply:
Ah, you're right. Thank you. Changed now.